nni/nas/oneshot/pytorch/supermodule/base.py

```diff
@@ -104,3 +126,21 @@ def mutate(cls, module: nn.Module, name: str, memo: dict[str, Any], mutate_kwarg
             See :class:`BaseOneShotLightningModule <nni.retiarii.oneshot.pytorch.base_lightning.BaseOneShotLightningModule>` for details.
         """
         raise NotImplementedError()
+
+    def _slice_params_mapping(self):
```

Comment:
Remove underline prefix if you want to make them interface.

Please add docstring.